build/Dockerfile.ci

```diff
@@ -138,28 +146,21 @@ ENV OPERATOR=/usr/local/bin/windows-machine-config-operator \
 
 # Changes needed for our CI
 
-# install make package
-RUN microdnf -y install make tar gzip wget && microdnf clean all
+# Install make package
+RUN microdnf -y install make && microdnf clean all
```

Comment:
I suggest leaving `tar` here for parity with the CPaaS Dockerfile. We need `tar` for `oc cp` to work in `render_templates`.

Reply:
in the second commit, I'm removing this part entirely, as I'm switching out ubi-minmal with the builder image, which should have tar included. I'll still add it back here in the first commit to make them idempotent